src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
 // write code here
+function sortList(list) {
+  const children = Array.from(list.children);
+
+  children.sort(sortFunc);
+
+  for (const obj of children) {
+    list.appendChild(obj);
+  }
+}
+
+function sortFunc(a, b) {
+  const aSal = formatSalary(a.dataset.salary);
```

Comment:
In `sortFunc`, the `formatSalary` function is called to parse the salary. Ensure that the `data-salary` attribute is correctly formatted as a string starting with a currency symbol followed by numbers.